substra/sdk/backends/local/dal.py

```diff
@@ -79,8 +82,36 @@ def login(self, username, password):
         if self._remote:
             self._remote.login(username, password)
 
-    def add(self, asset, exist_ok: bool = False):
-        return self._db.add(asset, exist_ok=exist_ok)
+    def add(self, asset):
+        return self._db.add(asset)
+
+    def check_exist(self, type_, key: str, exist_ok: bool = False):
```

Comment:
This function could be removed. It seems unused after the refactoring.

Reply:
Indeed, my melting brain forgot it completely 🔥